apps/trading/e2e/tests/spot_market/test_spot.py

```diff
@@ -0,0 +1,222 @@
+        ["Name", "Bitcoin / Tether USD (Spot)"],
+        ["Status", "Active"],
+        ["Trading Mode", "Continuous"],
+        ["Market Decimal Places", "1"],
+        ["Position Decimal Places", "2"],
+        ["Settlement Asset Decimal Places", "5"], # need to be remove after resolve this one  #6183
+        [" Tick Size", "1"]
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_instrument(page: Page):
+    page.get_by_test_id(market_title_test_id).get_by_text("Instrument").click()
+    fields = [
+        ["Market Name", "Bitcoin / Tether USD (Spot)"],
+        ["Code", "BTC/USDT-SPOT"],
+        ["Product Type", "Spot"],
+        ["Quote Name", "USDT"],
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_base_asset(page: Page, vega: VegaServiceNull):
+    page.get_by_test_id(market_title_test_id).get_by_text(
+        "Base asset").click()
+    tdai_id = vega.find_asset_id("BTC")
+    tdai_id_short = tdai_id[:6] + "…" + tdai_id[-4:]
+    fields = [
+        ["ID", tdai_id_short],
+        ["Type", "Builtin asset"],
+        ["Name", "BTC"],
+        ["Symbol", "BTC"],
+        ["Decimals", "5"],
+        ["Quantum", "0.00001"],
+        ["Status", "Enabled"],
+        ["Max faucet amount", "10,000,000,000.00"],
+        ["Infrastructure fee account balance", "0.003"],
+        ["Global reward pool account balance", "0.00"],
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_quote_data(page: Page, vega: VegaServiceNull):
+    page.get_by_test_id(market_title_test_id).get_by_text(
+        "Quote asset").click()
+    tdai_id = vega.find_asset_id("USDT")
+    tdai_id_short = tdai_id[:6] + "…" + tdai_id[-4:]
+    fields = [
+        ["ID", tdai_id_short],
+        ["Type", "Builtin asset"],
+        ["Name", "USDT"],
+        ["Symbol", "USDT"],
+        ["Decimals", "5"],
+        ["Quantum", "0.00001"],
+        ["Status", "Enabled"],
+        ["Max faucet amount", "10,000,000,000.00"],
+        ["Infrastructure fee account balance", "0.003"],
+        ["Global reward pool account balance", "0.00"],
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_metadata(page: Page):
+    page.get_by_test_id(market_title_test_id).get_by_text("Metadata").click()
+
+def test_market_spot_info_price_monitoring_bounds(page: Page):
+    page.get_by_test_id(market_title_test_id).get_by_text(
+        "Price monitoring bounds"
+    ).click()
+    expect(page.get_by_test_id(market_accordion_content).locator(
+        ".w-full").nth(1)).to_contain_text("90.001")
+    expect(page.get_by_test_id(market_accordion_content).locator(".w-full").nth(2)
+           ).to_contain_text("within 15m")
+    expect(page.get_by_test_id(market_accordion_content).locator(".text-left").nth(0)
+           ).to_contain_text("104.7")
+    expect(page.get_by_test_id(market_accordion_content).locator(".text-right").nth(0)
+           ).to_contain_text("110.3")
```

Comment:
Complicates things, but do you think its worth adding the test ids we need into the frontend so that we dont need to use .w-full etc to grab what we need.